src/lib/evil/evil_time.c

```diff
@@ -8,6 +8,7 @@
 #include <time.h>
 
 #include "evil_private.h"
+#include <evil_api.h>
```

Comment:
`evil_private.h` already includes `evil_api.h` :) So no need to include it again.

Reply:
Tks

Reply:
This doesn't seem to be resolved yet.